### PR TITLE
Add some blurb about the sprints. Closes #94

### LINKED ISSUE #94: Add some blurb about the sprints
We could do with some words about what the sprints are all about. I'll borrow last year's words.

### PR DESCRIPTION
**Pull request by rwillmer**

### CODE REVIEW

File: _pages/schedule.md [hunk 1]
```diff
@@ -22,3 +22,17 @@ The conference will take place from 9AM to 6PM.
 
 <a class="button" href="{{ site.schedule_link }}" target="_blank">View schedule!</a>
 
+## Sprints
+
+So you have listened to all the exciting topics on Django for three days and now feel motivated do something about it. What could be a better motivation than contributing to the very open source software you just listened to?
+
+On Thursday and Friday, you have the opportunity to do exactly that. This is your opportunity to contribute to Open Source software. Even if you haven't contributed before, this is also an option to take your first steps. In the sprint, you can contribute to Django as you see fit, and if you need help, there will be a pool of experienced contributors to guide you. You can also reach out through slack in the sprints channel.
```
Comment (rebkwok) [inline _pages/schedule.md:29]:
```suggestion
On Thursday and Friday, you have the opportunity to do exactly that. This is your opportunity to contribute to Open Source software. Even if you haven't contributed before, this is also an option to take your first steps. In the sprint, you can contribute to Django projects as you see fit, and if you need help, there will be a pool of experienced contributors to guide you. You can also reach out through Discord in the sprints channel.
```
I don't know for sure that there will be people sprinting on Django itself

File: _pages/schedule.md [hunk 2]
```diff
@@ -22,3 +22,17 @@ The conference will take place from 9AM to 6PM.
...
+
+<!--
+Thursday - 9AM to 6PM
+Friday - 9AM to 6PM
+-->
+
+The Sprints are free for all conference participants. However, as the Sprints venue does not have unlimited capacity, and we would like to prepare a precise amount of food and drinks, please keep your participation status updated on the ticketing website (after buying your ticket).
```
Comment (rebkwok) [inline _pages/schedule.md:36]:
```suggestion
The Sprints are free for all conference participants. However, in order for us to ensure we have catered for appropriate numbers, please keep your participation status updated on the ticketing website (after buying your ticket).
```
Made this a bit vaguer; we aren't providing drinks, and although the venue doesn't have unlimited capacity, I don't tknow what we do if we have more attendees than we've assumed (but it's very unlikely so I'm not worried :))

File: _pages/schedule.md [hunk 3]
```diff
@@ -22,3 +22,17 @@ The conference will take place from 9AM to 6PM.
...
 
+## Sprints
+
+So you have listened to all the exciting topics on Django for three days and now feel motivated do something about it. What could be a better motivation than contributing to the very open source software you just listened to?
+
+On Thursday and Friday, you have the opportunity to do exactly that. This is your opportunity to contribute to Open Source software. Even if you haven't contributed before, this is also an option to take your first steps. In the sprint, you can contribute to Django as you see fit, and if you need help, there will be a pool of experienced contributors to guide you. You can also reach out through slack in the sprints channel.
+
+<!--
+Thursday - 9AM to 6PM
+Friday - 9AM to 6PM
```
Comment (rebkwok) [inline _pages/schedule.md:33]:
```suggestion
Friday - 10AM to 5PM
```

File: _pages/schedule.md [hunk 4]
```diff
@@ -22,3 +22,17 @@ The conference will take place from 9AM to 6PM.
...
 <a class="button" href="{{ site.schedule_link }}" target="_blank">View schedule!</a>
 
+## Sprints
+
+So you have listened to all the exciting topics on Django for three days and now feel motivated do something about it. What could be a better motivation than contributing to the very open source software you just listened to?
+
+On Thursday and Friday, you have the opportunity to do exactly that. This is your opportunity to contribute to Open Source software. Even if you haven't contributed before, this is also an option to take your first steps. In the sprint, you can contribute to Django as you see fit, and if you need help, there will be a pool of experienced contributors to guide you. You can also reach out through slack in the sprints channel.
+
+<!--
+Thursday - 9AM to 6PM
```
Comment (rebkwok) [inline _pages/schedule.md:32]:
```suggestion
Thursday - 10AM to 5PM
```

**Review by rebkwok — COMMENTED** [review]
I think this would be better on its own page, linked under the "About" section in the nav. It isn't really schedule, apart from the days/times.

File: _pages/schedule.md (hunk 1 above)
Comment (rwillmer) [reply _pages/schedule.md:29]: What will they be doing? I've never been to one

**Review by rebkwok — APPROVED** [review]
Looks good! Only suggestion would be to remove the last bit from the Sprints page (where it says "We're still refining the schedule" for each day); we can add it back if we decide to put any sprint schedule info there, but it's usually a bit too ad-hoc for that, and it'd require someone to make website changes during the conference, which I'm not sure we'll have time for

File: _pages/schedule.md (hunk 1 above)
Comment (rebkwok) [reply _pages/schedule.md]: @rwillmer Sorry, I missed this question. Sprints are sessions of people working together on open source projects (or docs, in some cases).  The actual projects will be proposed at the conference, usually by people who are the main maintainer(s). I think often Django core developers will sprint on Django itself at a Django Con, but I don't want to say that on the website without knowing it for sure.

There's an intro talk I've seen Daniele do - I can't find the video right now, but this is a good description, which we should probably steal and put in the pretalx schedule:
https://pretalx.com/pyconuk-2019/talk/K973NJ/

**Pull request merged**